source/_posts/2016-12-03-remote-websockets-sonarr.markdown

```diff
@@ -0,0 +1,174 @@
+
+## {% linkable_title All changes  %}
+
+- Sensor: [Broadlink][boradlink] RM2 and A1 E-air support ([@skyval])
+- New services and improved device support for [HomeMatic][homematic] ([@pvizeli], [@danielperna84])
+- Device tracker: New support for [GPSLogger][gpslogger] ([@dainok])
+- Sensor: Support for [Sonarr][sonarr] ([@hborawski])
+- Sensor: [World Air Quality Index][waqi] sensor ([@valentinalexeev], [@fabaff])
+- Sensor: Support for [Dutch Smart Meter Requirements][dsmr] ([@aequitas])
+- Switch: [Hook][hook] support by hooksmarthome.com ([@dasos])
+- Camera: Integration for [Nest cameras][nest-cam] ([@technicalpickles])
+- Light: Support for light effects ([@Diaoul])
+- Sensor: New [Threshold][threshold] sensor ([@fabaff])
+- Media player: New [DuneHD][dunehd] integration([@valentinalexeev])
+- Media player: Controlling support for [Philips TVs][philips] ([@aequitas])
+- Camera: Support for [Amcrest][amcrest] cameras ([@tchellomello])
+- Sensor: Monitoring support for [Network UPS Tools (NUT)][nut] ([@mezz64])
+
+- Mediap player - Denon: Source selection support ([@Gilles95])
+- Sensor - MinMax: Precision now configurable ([@exxamalte])
+- Tellstick: Massive [improvement][tellstick] ([@magicus])
+- Light - Osram: New requirement ([@tfriedel])
+- Sensor - Systemmonitor: Support for removable network adapters ([@mnoorenberghe])
+- LiteJet: New trigger option ([@joncar])
+- Media player - Bose: Add Bose SoundTouch device support ([@CharlesBlonde])
+- HTTP: Re-organisation and [websockets] support ([@balloob])
+- HTTP: Advanced [IP filtering][filtering] ([@vkorn])
+- Sensor - KNX: Fix unit of mesaurement ([@cyberjunky])
+- Climate: New precision properties ([@sdague])
+- Sensor - TEMPer: Reset [devices][temper] on address change ([@vemek])
+- Core: Color names now follows w3.org recommandations ([@srcLurker])
+- Updater: Robustness improvements ([@balloob]])
+- Media player - MPD: Reconnect to daemon ([@janLo])
+- Device tracker: Fall-back for MAC address lookup ([@aequitas])
+- Sensor - Efergy: Get the amount of [energy consumed][efergy] ([@gonzalezcalleja])
+- Zeroconf: Fix for IPv6 support ([@rcloran])
+- Minor and not so minor features and bug fixes by [@turbokongen], [@sdague], [@pvizeli], [@fabaff], [@chapple], [@mweinelt], [@Khabi], [@balloob], [@mnestor], [@kellerza], [@Morrisai],
+[@michaelarnauts], [@tchellomello], [@lwis], [@bjarniivarsson], [@danielperna84], [@LinuxChristian], [@MartinHjelmare], [@dethpickle], [@jnewland], [@lichtteil], [@brandonweeks], [@partofthething], [@mnoorenberghe], [@bah2830], and [@albertoarias].
+
+## {% linkable_title Breaking changes %}
+
+- The [HomeMatic][homematic] component now uses a different syntax for hosts and the `set_value` service has been renamed.
+- All [RFXtrx][rfxtrx] sensors will get a new entity ID.
+- If you are using NGINX, you will have to [adapt your configuration][nginx].
```

Comment:
How does it need to be adapted? The ecosystem outlines how to use it in general, but doesn't indicate what changed if you were previously using nginx. I am running into this, and it was asked on https://home-assistant.io/blog/2016/12/03/remote-websockets-sonarr/#disqus_thread too